src/modules/ModalInterface.js

```diff
@@ -0,0 +1,158 @@
+export default class Modal {
+  static async getComments(showIndex) {
+    const baseUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/06PVGgvDd8phHuVhPNDL/comments?item_id=${showIndex}`;
+    const commentsResponse = await fetch(baseUrl, {
+      headers: {
+        'Content-type': 'application/json',
+      },
+    }).then((response) => {
+      if (response.status >= 400 && response.status < 600) {
+        const result = [{
+          username: '',
+          creation_date: '',
+          comment: 'No comments Yet',
+        }];
+        return result;
+      }
+      return response.json();
+    });
+
```

Comment:
**Changes Required**
- [ ] Fair effort on using the `async await`  promise syntactic sugar. However, you're mixing two syntax here, and the `async` keyword in this context is redundant. May we please remove the `.then` and stick to the `async await` format.

Reply:
Changes made.